RecoLocalFastTime/FTLClusterizer/src/MTDThresholdClusterizer.cc

```diff
@@ -74,7 +74,7 @@ bool MTDThresholdClusterizer::setup(const MTDGeometry* geom, const MTDTopology*
 
   if (nrows > theBuffer.rows() || ncols > theBuffer.columns()) {  // change only when a larger is needed
     // Resize the buffer
-    theBuffer.setSize(nrows + 1, ncols + 1);  // +1 needed for MTD
+    theBuffer.setSize(nrows, ncols);
```

Comment:
Just out of curiosity, why isn't "+1 needed for MTD" anymore?

Reply:
I would be curious to know why it was in first place, because I failed to understand it. And with this "+1" you add a fake additional element to the buffer, for what I could see in my tests. I looked back at the code history and I did not see any explicit comment about this, I was the release manager at that time and I recall this code was frantically reviewed and developed under pressure, so not everything was optimal. Anyway if you see a reason to leave "+1" please explain it to me.